### ISSUE TITLE
Error running custom generator

### ISSUE THREAD

**Issue by jimcullenaus** [swagger-api/swagger-codegen#7794]
##### Description

Running through the steps [in this doc](https://github.com/swagger-api/swagger-codegen#making-your-own-codegen-modules) for how to create a custom generator. I've created the basic sample project in `output/myLibrary`, compiled it with `mvn package`, although I had to add

```xml
<repositories>
    <repository>
        <id>swagger-codegen-repo</id>
        <url>file://${basedir}/swagger-codegen-repo</url>
    </repository>
</repositories>
```

to the pom.xml file (as well as possibly other minor changes that were necessary) in order to get `mvn package` to compile successfully. The `io.swagger.swagger-modules` dependency also had to be added in order to get to the stage I am currently at.

I have made no changes to the `MyclientcodegenGenerator.java` file.

(or the equivalent modified for Windows), the output is 

##### Swagger-codegen version

This occurs on both Windows (running in Power Shell) and Linux (Ubuntu Bash through WSL), with the latest version of swagger-codegen.

##### Command line used for generation

When I execute

```shell
java -cp output/myLibrary/target/myClientCodegen-swagger-codegen-1.0.0.jar:modules/swagger-codegen-cli/target/swagger-codegen-cli.jar \
  io.swagger.codegen.SwaggerCodegen generate -l myClientCodegen\
  -i http://petstore.swagger.io/v2/swagger.json \
  -o myClient
```

the output is 

```
Exception in thread "main" java.lang.RuntimeException: Could not generate supporting file 'SupportingFile:
        templateFile: myFile.mustache
        folder:
        destinationFilename: myFile.sample
```

Using the `-jar` method described elsewhere results in

```
Exception in thread "main" io.airlift.airline.ParseArgumentsUnexpectedException: Found unexpected parameters: [io.swagger.codegen.SwaggerCodegen, generate, -l, myClientCodegen, -i, http://petstore.swagger.io/v2/swagger.json, -o, myClient]
```

##### Related issues/PRs

Appears to be very similar to #1566, the error that @xhh was having right at the end.

##### Suggest a fix/enhancement

It seems as if `myFile.mustache` is not being produced and/or placed in the right location by `swagger-codegen-cli.jar meta`, or that the `README` produced (and the guide on GitHub) do not adequately explain the need to create a `myFile.mustache` file and how to go about that.

Thanks in advance.

**Comment by jmini** [comment]
I don't think that the bootstrap project generated by the engine is up to date with all the changes made with version `3.0.0-SNAPSHOT` of `swagger-codegen`. See also #7845

**Comment by jimcullenaus** [comment]
This issue was actually experienced when trying to work with the 2.4.0-SNAPSHOT version. It was only after I eventually (somehow — I don't recall how exactly) got around this issue that I moved on to trying 3.0.0 as in issue 7845.

**Comment by anselmos** [comment]
So I've currently was able to generate files with : 

```
java -cp modules/swagger-codegen//target/-swagger-codegen-3.0.10.jar:modules/swagger-codegen-cli/target/swagger-codegen-cli.jar:lib/swagger-codegen-generators-1.0.11-SNAPSHOT.jar \
  io.swagger.codegen.v3.cli.SwaggerCodegen  generate  -l io.swagger.codegen.v3.generators.python.PythonFlaskConnexionCodegen \
  -i http://petstore.swagger.io/v2/swagger.json \
  --template-engine "mustache" \
  -o /tmp/pythontest
```

Where  `lib/swagger-codegen-generators-1.0.11-SNAPSHOT.jar` is the packaged swagger-codegen-genrators with modified files for in this example python-flask .

Hopefully someone will be able to reuse this.
